app/tests/unit/test_openstack_builder.py

```diff
@@ -669,23 +669,24 @@ def test__generate_cloud_init_script(
 
 function configure_proxy() {{
     local proxy="$1"
-    if [[ -z "$proxy" ]]; then
-        return
-    fi
+
+    echo "Installing aproxy"
+    # We always want snap aproxy and nft to be installed, even it they are not used by the image builder.
```

Comment:
We are not installing nft in the lines below? So I think we shouldn't mention it, otherwise its confusing.